addon/components/structure-plugin/_private/structure.gts

```diff
@@ -95,12 +95,17 @@ export default class Structure extends Component<Sig> {
     return this.node.attrs.structureType as StructureType;
   }
 
+  get fullLenghtArticles() {
+    return this.node.attrs.fullLenghtArticles as boolean;
```

Comment:
Very pedantic, but this should be `fullLengthArticles`

Reply:
oh good catch, not pedantic at all, the typo is also in the config you pass in so this will trip up users

Reply:
oh sorry brainfarted a bit there